### DISCUSSION TITLE
Getting imagery -- especially in 360 format? -- from Stellarium for outside use?

### DISCUSSION THREAD

**Discussion by clcErin** [Stellarium/stellarium#2337] [Q&A]
Is there a way to access/create stand-alone image files based on what is visible in Stellarium?
I'm in education, and I would like students to be able to mark what they see in the stellarium sky on paper, with the greek mythology artwork as a reference. (Ex: It's easier for a student to say "Mars is just above Virgo's knee in the picture" than to say "Mars is 30 arc seconds azimuthily above gamma virginius" or something.)
Eventually, I want to make a project where the zodiacal constellations are flattened out onto a ring, and students can mark where Earth is in its orbit on their birthday based on where they observe the sun using Stellarium.
Right now my only option is to take a lot of screen shots and manual stitch them together. I know NOTHING about the back end of stellarium, I just use it from the front end a little.
Thanks for any tips/ideas!
PS- I have a planetarium that runs Nightshade as well, if it makes more sense to go that route.

**Comment by gzotti** [comment]
Years ago I added the skybox script. Its result images can be stitched in some panorama software like Hugin.

**Reply by clcErin** [reply]
Thanks! I'll look into this. Rookie question that I can probably find the answer to, but that you may be able to direct me to sooner: How does one add in scripts? Would I start with the "regular" version of Stellarium I downloaded from the website for Windows, or do I have to go about it a different way?

**Reply by gzotti** [reply]
The skybox script is part of the regular distribution. Running scripts is documented in the User Guide.
